README.md

```diff
@@ -1,15 +1,34 @@
+I certify that I have listed all sources used to complete this exercise, including the use of any Large Language Models. All of the work is my own, except where stated otherwise. I am aware that plagiarism carries severe penalties and that if plagiarism is suspected, charges may be filed against me without prior notice.
+No sources were used.
 # Theory vs. Practice
 
 - List 3 reasons why asymptotic analysis may be misleading with respect to
   actual performance in practice.
+  - Asymptotic analysis finds a complexity that applies for all values >= n so it doesn't tell us anything about time complexity for input sizes less than n.
```

Comment:
Do you mean $n_0$ here?

Reply:
Yes, thats exactly what I meant. Its been changed now.